plugins/net_plugin/net_plugin.cpp

```diff
@@ -1683,7 +1699,7 @@ namespace eosio {
             if (!b) {
                send_req = true;
                req.req_blocks.ids.push_back( blkid );
-               req_blks.push_back( blkid );
+               req_blks.push_back( (block_request){blkid, generated} );
```

Comment:
I think this would look cleaner without the typename: push_back( {blkid, generated} );